report/test/renderer/dom-test.js

```diff
@@ -153,4 +152,73 @@ describe('DOM', () => {
           '<code>more code</code>, and yet event <code>more</code>.');
     });
   });
+
+  describe('safelySetHref', () => {
+    it('sets href for safe destinations', () => {
+      [
+        'https://safe.com/',
+        'http://safe.com/',
+      ].forEach(url => {
+        const a = dom.createElement('a');
+        dom.safelySetHref(a, url);
+        expect(a.href).toEqual(url);
+      });
+    });
+
+    it('sets href for safe in-page named anchors', () => {
+      const a = dom.createElement('a');
+      dom.safelySetHref(a, '#footer');
+      expect(a.href).toEqual('about:blank#footer');
+    });
+
+    it('doesnt set href if destination is unsafe', () => {
+      [
+        'javascript:evil()',
+        'data:text/html;base64,abcdef',
+        'data:application/json;base64,abcdef',
+        'ftp://evil.com/',
+        'blob:http://example.com/ca6df701-9c67-49fd-a787',
+        'intent://example.com',
+        'filesystem:http://localhost/img.png',
+        'ws://evilchat.com/',
+        'wss://evilchat.com/',
+        'about:about',
+        'chrome://chrome-urls/',
+      ].forEach(url => {
+        const a = dom.createElement('a');
+        dom.safelySetHref(a, url);
+
+        expect(a.href).toEqual('');
+        expect(!a.href).toBeTruthy();
```

Comment:
what is the purpose of this particular assertion?

Reply:
no good reason. removed.